### ISSUE TITLE
Pagination suffix work change the href behaviour?

### ISSUE THREAD

**Issue by Geobert** [cobalt-org/cobalt.rs#666]
I've noticed something today as I've recompiled a cobalt at work. My pagination navigation is like https://github.com/Geobert/blog/blob/master/src/_includes/nav_pagination.liquid

I'm using `"/{{ paginator.previous_index_permalink }}"` in `href`.
Before https://github.com/cobalt-org/cobalt.rs/pull/631, when serving the blog, I got `http://localhost:3000/all/_p/2/` as link. 
After: `all/_p/2/` which leads nowhere.

So basically, I broke it… 
Opening this issue to fix it ASAP :)

**Comment by Geobert** [comment]
Hm, it seems that I added a leading `/` so with my leading `/` in the `href` it breaks the link.

I don't know if it's a bug then…

**Comment by Geobert** [comment]
according to https://github.com/cobalt-org/cobalt.rs/issues/622 the leading slash is expected.

**Comment by Geobert** [comment]
After a night of sleep, maybe even if we agreed on this result, the leading slash in `paginator.previous_index_permalink` is not a good idea for consistency sake:

for my about page, I have `href="/about"`, for next/previous post we have `href=/{{page.previous.permalink}}`.

So it's a bit odd I have to remove the slash in href for pagination case.

I'll fix that ASAP :)

**Comment by epage** [comment]
Yes, the main thing is to be consistent with regular pagination links.
